### PR TITLE
docs: Add 1.3.0 release note

### PR DESCRIPTION
**Pull request by jholdstock**

### CODE REVIEW

File: docs/release-notes/release-notes.1.3.0.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+- Fee calculation now takes the new block reward subsidy split from the activation
+  of [DCP-0012](https://github.com/decred/dcps/blob/master/dcp-0012/dcp-0012.mediawiki)
+  into consideration. In practice, this means that VSPs will begin charging
+  marginally higher fees.
+- vspd will now distinguish between tickets which are "missed" and tickets which
+  are "expired". Previously these tickets would be considered as a single set
+  labelled "expired". This is acheived using dcrd and
+  [Golomb-Coded Set filters](https://github.com/decred/dcrd/tree/master/gcs#gcs).
+  vspd 1.3.0 will perform a **one-time update of every revoked ticket in the
```
Comment (davecgh) [inline docs/release-notes/release-notes.1.3.0.md:45]:
```suggestion
-  vspd 1.3.0 will perform a **one-time update of every revoked ticket in the
```

File: docs/release-notes/release-notes.1.3.0.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  database** the first time it is started. This may take a while for VSPs which
+  have been active for a long time or have a large number of revoked tickets.
+- Expired and missed tickets added to `/vspinfo`. Revoked is now deprecated.
+- Minor improvements to web UI:
+  - Homepage now displays expired and missed tickets instead of revoked tickets.
+  - Homepage no longer displays the current network unless it is running on
```
Comment (davecgh) [inline docs/release-notes/release-notes.1.3.0.md:51]:
```suggestion
  - Homepage no longer displays the current network unless it is not running mainnet
```

File: docs/release-notes/release-notes.1.3.0.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    testnet or simnet.
+  - Admin page now displays decoded transactions in a human readable format
+    rather than just the raw bytes.
```
Comment (davecgh) [inline docs/release-notes/release-notes.1.3.0.md:54]:
```suggestion
    in addition to just the raw bytes.
```

**Comment by jholdstock** [comment]
Thanks @davecgh, pushed an updated commit with your suggestions.

File: docs/release-notes/release-notes.1.3.0.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+## Notable Changes
+
+- Fee calculation now takes the new block reward subsidy split from the activation
+  of [DCP-0012](https://github.com/decred/dcps/blob/master/dcp-0012/dcp-0012.mediawiki)
+  into consideration. In practice, this means that VSPs will begin charging
+  marginally higher fees.
+- vspd will now distinguish between tickets which are "missed" and tickets which
+  are "expired". Previously these tickets would be considered as a single set
+  labelled "expired". This is acheived using dcrd and
```
Comment (davecgh) [inline docs/release-notes/release-notes.1.3.0.md:43]: Not sure if you aim for American or British English, but one l for American.  Doesn't really matter, but probably best to be consistent with whichever style is chosen.

**Review by davecgh — APPROVED** [review]
Approving this since it looks good regardless.  Your choice on whether to adopt American vs British.

**Pull request merged**